src/python/py/models/builder.py

```diff
@@ -347,6 +347,7 @@ def make_genai_config(self, model_name_or_path, extra_kwargs, out_dir):
                     "num_key_value_heads": self.num_kv_heads,
                 },
                 "eos_token_id": config.eos_token_id,
+                "original_context_length": self.original_context_length,
```

Comment:
Should we call it `original_max_position_embeddings` in out genai config file as well?

Reply:
Not sure why we changed the name to original_context_length, it seems to be used in more scenarios than just short and long factor. Maybe @kunal-vaishnavi can provide more context?

Reply:
I am reverting this now, will discuss this with Kunal once he is back. Will merge the PR for now.